Lib/_collections_abc.py

```diff
@@ -67,6 +67,10 @@
 del _ag
 
 
+# needed for making various ABCs generic
+GenericAlias = type(list[str])
```

Comment:
I would import it from types -- by the time this module is imported, types has already been imported (at least by `collections.__init__`).